lib/mayaUsd/ufe/Global.cpp

```diff
@@ -120,6 +121,12 @@ void mayaStopWaitCursor() { MGlobal::executeCommand("waitCursor -state 0"); }
 // Note: MayaUsd::ufe::getStage takes two parameters, so wrap it in a function taking only one.
 PXR_NS::UsdStageWeakPtr mayaGetStage(const Ufe::Path& path) { return MayaUsd::ufe::getStage(path); }
 
+// Wrapped to return std::string from static function returning const std::string.
+std::string defaultMaterialsScopeName()
+{
+    return UsdMayaJobExportArgs::GetDefaultMaterialsScopeName();
+}
```

Comment:
This wrapper is used below when setting the DCC function. So MayaUsd behaves the same as before.